### PR TITLE
Removes redundant section.

### PR DESCRIPTION
**Pull request by bycdiaz**
Removes section on motivation. The following lesson deals entirely with motivation. In addition, the link to article mentioned is broken and the post doesn't seem to be present on the internet at another address.

Resolves issue #12374.

### CODE REVIEW

File: web_development_101/the_basics/introduction_to_web_development.md [hunk 1]
```diff
@@ -50,17 +50,6 @@ These are some of the basic tools you will use regularly. You may not know what
...
 
-### Motivation
-
-Learning to code is incredibly rewarding but can also be difficult and frustrating.
-The strongest assets you can have as a student are a desire to build, a problem-solving mind, and persistence in the face of setbacks.
-
-The web development industry has a long history of successful developers with varying backgrounds, so people tend to care more about what you've actually **built** than how you got there.
-
-Once you start to feel comfortable with the tools, start hacking on open source projects [(like The Odin project itself)](/contributing). The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
-
```
Comment (leila-alderman) [inline web_development_101/the_basics/introduction_to_web_development.md]: These two paragraphs aren't really about motivation and are worth preserving. I would recommend simply relocating these two paragraphs to the bottom of the `Why Odin?` section below.

File: web_development_101/the_basics/introduction_to_web_development.md [hunk 2]
```diff
@@ -50,17 +50,6 @@ These are some of the basic tools you will use regularly. You may not know what
 * **[Git](https://git-scm.com/)**
 * **[GitHub](https://github.com/)**
 
-### Motivation
-
-Learning to code is incredibly rewarding but can also be difficult and frustrating.
-The strongest assets you can have as a student are a desire to build, a problem-solving mind, and persistence in the face of setbacks.
```
Comment (leila-alderman) [inline web_development_101/the_basics/introduction_to_web_development.md]: I've moved this text over to the next lesson, "Gearing Up". 

https://github.com/TheOdinProject/curriculum/pull/12574

**Comment by I3uckwheat** [comment]
bump

**Comment by leila-alderman** [comment]
@bycdiaz I had left a comment previously about relocating some of the text to another section instead of deleting all of the text. Once that's been done, this should be good to go.

File: web_development_101/the_basics/introduction_to_web_development.md (hunk 1 above)
Comment (bycdiaz) [reply web_development_101/the_basics/introduction_to_web_development.md]: Cool. Just moved it.

**Pull request merged**